src/main/grammars/RustLexer.flex

```diff
@@ -228,7 +228,10 @@ OUTER_EOL_DOC = ({EOL_DOC_LINE}{EOL_WS})*{EOL_DOC_LINE}
   "/*"                            { yybegin(IN_BLOCK_COMMENT); yypushback(2); }
 
   {INNER_EOL_DOC}                 { return INNER_EOL_DOC_COMMENT; }
-  {OUTER_EOL_DOC}                 { return OUTER_EOL_DOC_COMMENT; }
+  {OUTER_EOL_DOC} / [^\/]         { //fix lookahead on EOF
+                                    if (zzMarkedPos + 1 == zzBuffer.length())
```

Comment:
Can you explain what you are trying to fix here?
Now `///<EOF>` is not considered part of the doc comment:

<img width="400" alt="1" src="https://user-images.githubusercontent.com/6079006/57783227-71a3a480-7736-11e9-9fad-cb88f9d21eb4.png">
<img width="400" alt="2" src="https://user-images.githubusercontent.com/6079006/57783258-7d8f6680-7736-11e9-82da-96615541c1ae.png">

Reply:
Problem I am trying to fix is that you can't add <EOF> as accepted lookahead symbol which causes last symbol not to be included into doc comment token. So I just explicitly add last symbol to token. But apparently it is not enough. Then I suppose I need to write new lexer state for line doc comments.